### PR TITLE
magento/devdocs#: Add mergeCarts query API errors

### PR DESCRIPTION
**Pull request by atwixfirster**
## Purpose of this pull request

This pull request (PR) adds information about the API errors of `mergeCarts` mutation.

## Affected DevDocs pages

- https://devdocs.magento.com/guides/v2.3/graphql/mutations/merge-carts.html

## Links to Magento source code

- [Current user does not have an active cart.](https://github.com/magento/magento2/blob/2.3.4/dev/tests/api-functional/testsuite/Magento/GraphQl/Quote/Customer/MergeCartsTest.php#L111)
- [Required parameter "destination_cart_id" is missing](https://github.com/magento/magento2/blob/2.3.4/app/code/Magento/QuoteGraphQl/Model/Resolver/MergeCarts.php#L56)
- [Required parameter "source_cart_id" is missing](https://github.com/magento/magento2/blob/2.3.4/app/code/Magento/QuoteGraphQl/Model/Resolver/MergeCarts.php#L52)
- [The current customer isn't authorized.](https://github.com/magento/magento2/blob/2.3.4/app/code/Magento/QuoteGraphQl/Model/Resolver/MergeCarts.php#L61)
- [The current user cannot perform operations on cart](https://github.com/magento/magento2/blob/2.3.4/dev/tests/api-functional/testsuite/Magento/GraphQl/Quote/Customer/MergeCartsTest.php#L144)

### CODE REVIEW

**Comment by devops-devdocs** [comment]
An admin must run tests on this PR before it can be merged.

File: src/guides/v2.3/graphql/mutations/merge-carts.md [hunk 1]
```diff
@@ -83,3 +83,13 @@ Attribute |  Data Type | Description
...
 [Cart query output]({{page.baseurl}}/graphql/queries/cart.html#cart-output) provides more information about the `Cart` object.
+
+## Errors
+
+Error | Description
+--- | ---
+`Current user does not have an active cart.` | `mergeCarts` mutation deactivates provided in the `source_cart_id` attribute guest's cart after merge, thus guest will be not able to make any further operations with it.  
+`Required parameter "destination_cart_id" is missing` | The `destination_cart_id` attribute contains an empty value.
+`Required parameter "source_cart_id" is missing` | The `source_cart_id` attribute contains an empty value.
+`The current customer isn't authorized.` | The current customer is not currently logged in, or the customer's token does not exist in the `oauth_token` table, or you tried to merge guests carts.
```
Comment (rogyar) [inline src/guides/v2.3/graphql/mutations/merge-carts.md]: > you tried to merge guests carts

Thats a very good point 👍

File: src/guides/v2.3/graphql/mutations/merge-carts.md [hunk 2]
```diff
@@ -83,3 +83,13 @@ Attribute |  Data Type | Description
 {% include graphql/cart-object.md %}
 
 [Cart query output]({{page.baseurl}}/graphql/queries/cart.html#cart-output) provides more information about the `Cart` object.
+
+## Errors
+
+Error | Description
+--- | ---
+`Current user does not have an active cart.` | `mergeCarts` mutation deactivates provided in the `source_cart_id` attribute guest's cart after merge, thus guest will be not able to make any further operations with it.
```
Comment (keharper) [inline src/guides/v2.3/graphql/mutations/merge-carts.md]:
```suggestion
`Current user does not have an active cart.` | The `mergeCarts` mutation deactivates the guest cart specified in the `source_cart_id` after merging. The guest cannot make any further operations with it.
```

File: src/guides/v2.3/graphql/mutations/merge-carts.md (hunk 1 above)
Comment (keharper) [inline src/guides/v2.3/graphql/mutations/merge-carts.md]:
```suggestion
`The current customer isn't authorized.` | The current customer is not currently logged in, or the customer's token does not exist in the `oauth_token` table, or you tried to merge two guest carts.
```

File: src/guides/v2.3/graphql/mutations/merge-carts.md [hunk 3]
```diff
@@ -83,3 +83,13 @@ Attribute |  Data Type | Description
...
+
+## Errors
+
+Error | Description
+--- | ---
+`Current user does not have an active cart.` | `mergeCarts` mutation deactivates provided in the `source_cart_id` attribute guest's cart after merge, thus guest will be not able to make any further operations with it.  
+`Required parameter "destination_cart_id" is missing` | The `destination_cart_id` attribute contains an empty value.
+`Required parameter "source_cart_id" is missing` | The `source_cart_id` attribute contains an empty value.
+`The current customer isn't authorized.` | The current customer is not currently logged in, or the customer's token does not exist in the `oauth_token` table, or you tried to merge guests carts.
+`The current user cannot perform operations on cart` | Authorized customer tries to merge a guest cart into a cart of another customer.
```
Comment (keharper) [inline src/guides/v2.3/graphql/mutations/merge-carts.md]:
```suggestion
`The current user cannot perform operations on cart` | The authorized customer tried to merge a guest cart into the cart of another customer.
```

Comment (atwixfirster) [reply src/guides/v2.3/graphql/mutations/merge-carts.md]: **fixed**

File: src/guides/v2.3/graphql/mutations/merge-carts.md (hunk 1 above)
Comment (atwixfirster) [reply src/guides/v2.3/graphql/mutations/merge-carts.md]: **fixed**

File: src/guides/v2.3/graphql/mutations/merge-carts.md (hunk 2 above)
Comment (atwixfirster) [reply src/guides/v2.3/graphql/mutations/merge-carts.md]: **implemented**

**Comment by atwixfirster** [comment]
@keharper all your suggestions have been implemented. Could you please review? Thank you!

**Pull request merged**